docs/rules/no-duplicate-imports.md

```diff
@@ -14,57 +16,47 @@ import { find } from 'module';
 
 This inspection reports any duplicated module in an import statement.
 
-The following patterns are considered problems:
+Example of **incorrect** code for this rule:
 
 ```js
 /*eslint no-duplicate-imports: "error"*/
 
 import { merge } from 'module';
-import path from 'another-module';
-import { find } from 'module';
-
-
-import { merge } from 'module';
+import something from 'another-module';
 import _, { find } from 'module';
 ```
 
-The following patterns are not considered problems:
+Example of **correct** code for this rule:
 
 ```js
 /*eslint no-duplicate-imports: "error"*/
 
-import { merge, find } from 'module';
-import path from 'another-module';
+import _, { merge, find } from 'module';
+import something from 'another-module';
 ```
 
 ## Options
 
 This rule takes one optional argument, an object with a single key, `includeExports` which is a `boolean`. It defaults to `false`.
 
-With this option set to `true`, the following patterns are considered problems:
+If re-exporting from an imported module, you should add the imports to the `import`-statement, and export that directly, not use `export ... from`.
+
+Example of **incorrect** code for this rule with the `{ includeExports: true }` option:
```

Comment:
The pattern we are moving toward in double quote marks around option property keys (as would be required in .estlintrc.json files and is allowed in .eslintrc.js files) in the sentence and eslint comment.

And the space inside the braces is according to the pattern.